lib/core/MongoDbOperationStore.ts

```diff
@@ -101,24 +99,41 @@ export default class MongoDbOperationStore implements IOperationStore {
    */
   public async delete (transactionNumber?: number): Promise<void> {
     if (transactionNumber) {
-      await this.collection!.deleteMany({ transactionNumber: { $gt: Long.fromNumber(transactionNumber) } });
+      await this.collection!.deleteMany({ txnNumber: { $gt: Long.fromNumber(transactionNumber) } });
     } else {
       await this.collection!.deleteMany({});
     }
   }
 
+  public async deleteUpdatesEarlierThan (didUniqueSuffix: string, transactionNumber: number, operationIndex: number): Promise<void> {
```

Comment:
I don't think we should change the method argument names, the MongoDb field names are shortened only due to DB compatibility issues, rest of the repo continues to use the non-abbreviated name.

I don't think linter complains about lack of comments in this case (are you saying your linter actually does?) because it's the corresponding public interface that should have the comments.